wayland/window.c

```diff
@@ -47,7 +47,17 @@ static void pointer_handle_button(void *data, struct wl_pointer *pointer, uint32
 }
 
 static void pointer_handle_axis(void *data, struct wl_pointer *pointer,
-		    uint32_t time, uint32_t axis, wl_fixed_t value) {
+				uint32_t time, uint32_t axis, wl_fixed_t value) {
```

Comment:
Not exactly sure what axis and value. Documentation is lacking. Testing suggests that on a mouse with a standard scroll wheel axis is zero and value is -10.0/10.0 when scrolled up/down.